### DISCUSSION TITLE
What is next after Grafana?

### DISCUSSION THREAD

**Discussion by IvanZhd** [pierosavi/pierosavi-imageit-panel#123] [General]
Dear Piero,
you mentioned that you don't use grafana anymore. Have you switch the job or you found a better solution for making dashboards and capable to add images on a dashboard? Thank you for your time and sharing your experience.

**Comment by pierosavi** [comment]
> Dear Piero,
> you mentioned that you don't use grafana anymore. Have you switch the job or you found a better solution for making dashboards and capable to add images on a dashboard? Thank you for your time and sharing your experience.

I switched job to another company :) Sorry, I don't have experience with any Grafana alternative
